app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsFragment.kt

```diff
@@ -90,6 +89,7 @@ class OwnerDetailsFragment : BaseViewBindingFragment<FragmentOwnerDetailsBinding
 
                         ownerQrCode.setImageBitmap(viewAction.ownerDetails.qrCode)
                         content.animate().alpha(1f).setDuration(1000)
+                        exportButton.isEnabled = viewAction.ownerDetails.exportable
```

Comment:
exportable would be false for hardware wallet backed keys?

Reply:
yes, there would be no private key or seed phrase stored on device